jekyll/_cci2/deployment_integrations.md

```diff
@@ -6,8 +6,33 @@ categories: [deploying]
 order: 10
 ---
 
+## Available Deployment Targets
 
-## Conditional Deployment 
+You can deploy to any service you choose with CircleCI 2.0. By adding commands to `.circleci/config.yml` and setting secrets on the 'Project Settings > Environment Variables' screen, you have a lot of flexibility. For deployment targets other than those listed in 'Deployment Integrations' below, please see the '<a href="#other-deployments">Other Deployments</a>' section.
+
+## Deployment Integrations
+
+**Note:** Currently (during Beta), the CircleCI 'Project Settings' UI has a section titled 'Continuous Deployment':
+
+![Deployment Integrations]({{ site.baseurl }}/assets/img/docs/deployment-integrations.png)
+
+### AWS Deployment
+
+1. To deploy to AWS from CircleCI 2.0 use the [awscli installation instructions](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) to ensure that `awscli` is installed in your primary container. 
+
+2. Add your AWS credentials to the **Project Settings > AWS Permissions** page in the CircleCI application.
```

Comment:
That info was already in the 2.0 docs so I presume it is correct and had been tested. Will double check.